gdo_voicebot/grammar_correction_service/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM python:3.7-slim
+
+ENV PYTHONIOENCODING=UTF-8
+
+RUN apt-get update && apt-get install -y \
+    ffmpeg espeak libespeak-dev python-dev build-essential\
```

Comment:
is ffmpeg espeak libspeak essential here? Mifu was using them for resampling in the emotion service but I guess you're not doing that.